HW-11/homework11(1).py

```diff
@@ -0,0 +1,20 @@
+import json
+from datetime import datetime
+
+
+def MyDecFunc(func):
+    def wrap():
+        time = datetime.now()
+        text = f'Open function {func} \n Time: {time}'
+        with open(r'C:\HW\HomeWork\HW-11\file_dec', 'x') as file_dec:
```

Comment:
Use related path here instead of absolute. My computer, for example, don't have this path and I can't use this. And use `w+` modifier to be able to use this decorator more than once.